src/chat_do.py

```
@@ -0,0 +1,406 @@
+            try:
+                raw_att = ws.deserializeAttachment()
+                if raw_att:
+                    att = json.loads(raw_att) if isinstance(raw_att, str) else raw_att
+                    sid = att.get("session_id")
+                    info = self.sessions.get(sid)
+            except Exception:
+                pass
+        if not info:
+            print("INFO IS NONE - EXITING")
+            return
+        else:
```

Ruff (0.16.0)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)